manim/mobject/mobject.py

```diff
@@ -499,13 +541,13 @@ def remove(self, *mobjects: "Mobject") -> "Mobject":
                 self.submobjects.remove(mobject)
         return self
 
-    def __sub__(self, other):
+    def __sub__(self, other: Mobject) -> Mobject:
```

Comment:
Lying about the return value

Reply:
Do you mean the return type is wrong? If so what should it be? It's pretty hard to tell in some cases!

I don't think you mean *lying* - "lying" means being deliberately deceptive and I hope you aren't accusing me of that!